.github/actions/setup-e2e-tools/action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'setup-e2e-tools'
+description: 'Set up required tooling for creating an e2e test environment'
+
```

Comment:
It might not be necessary now, but exposing some of the important action configuration options as inputs with sane defaults would go a long way to making this more useable. If nothing else, being able to pin to a specific version of `skaffold` or `minikube` would be good.